app/controllers/integrations/base_controller.rb

```diff
@@ -2,11 +2,18 @@ class Integrations::BaseController < ApplicationController
   skip_around_action :login_user
   skip_before_action :verify_authenticity_token
   before_action :validate_request
+  after_action :record_webhook
 
   def create
-    return head(:ok) unless deploy?
+    unless deploy?
+      record_log :error, "Request is not a deploy"
```

Comment:
changed to info

On Fri, Jun 10, 2016 at 12:47 PM, Jon Moter <notifications@github.com>
wrote:

> In app/controllers/integrations/base_controller.rb
> <https://github.com/zendesk/samson/pull/1060#discussion_r66670907>:
>
> >
> >    def create
> > -    return head(:ok) unless deploy?
> > +    unless deploy?
> > +      record_log :error, "Request is not a deploy"
>
> Not really an error. We'll probably get notifications for a bunch of stuff
> that shouldn't be a deploy, like a PR against a branch that isn't master.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1060/files/896a67860df898ea7045f3a2dec7402b939763a1#r66670907>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAsZ9fHQ8uvyns33op3jD2CQBosLGNzks5qKb9OgaJpZM4IzNBF>
> .
>